docs/sources/flow/reference/stdlib/coalesce.md

```diff
@@ -0,0 +1,21 @@
+---
+aliases:
+- ../../configuration-language/standard-library/coalesce/
+title: coalesce
+---
+
+# coalesce
+
+`coalesce` takes any number of arguments and returns the first one that isn't null, an empty string, empty list or empty object. It can
+be combined with other functions to provide a default.
```

Comment:
Since we added that it will always return the last element even if null, then we should document that here.